website/content/docs/drivers/raw_exec.mdx

```diff
@@ -57,6 +57,9 @@ the Nomad client has been hardened according to the [production][hardening] guid
   must be an absolute path. This will also change the working directory when
   using `nomad alloc exec`.
 
+- `denied_envvars` - (Optional) Passes a list of environment variables to be
+  scrubbed from the task environment. Supports simple globbing, "*" wildcard
+  accepted as prefix and/or suffix.
```

Comment:
```suggestion
- `denied_envvars` - (Optional) Passes a list of environment variables that
  the driver should scrub from all task environments. Supports globbing "*",
  with wildcard accepted as prefix and/or 
```

style: use active voice; also we don't use words like "simple" so I removed it rather than substitute "basic"

I'm not sure if the driver scrubs the variables or Nomad scrubs the variables